src/test/packages.test.ts

```diff
@@ -0,0 +1,62 @@
+import { ok, strictEqual } from "assert";
+import * as Path from "path";
+
+import * as td from "..";
+import { Logger } from "../lib/utils";
+import { expandPackages } from "../lib/utils/package-manifest";
+
+describe("Packages support", () => {
+    it("handles monorepos", () => {
+        const base = Path.join(__dirname, "packages", "ts-monorepo");
+        const app = new td.Application();
+        app.options.addReader(new td.TypeDocReader());
+        app.bootstrap({
+            options: Path.join(base, "typedoc.json"),
+        });
+        const project = app.convert();
+        ok(project, "Failed to convert");
+        const result = app.serializer.projectToObject(project);
+        ok(result.children !== undefined);
+        strictEqual(
```

Comment:
The validation I'm doing in this and other cases is pretty minimal atm, I mainly wanted to focus on the new behaviour around discovering packages as I believe most other behaviour is already tested elsewhere. LMK if there are other things you believe we should check here though.